chaoscenter/authentication/api/middleware/coverage.html

```diff
@@ -0,0 +1,142 @@
+
```

Comment:
@nikzayn you can remove both the coverage files, you don't need to commit those generated files.